dev/tasks/r/github.packages.yml

```
@@ -297,7 +312,7 @@ jobs:
         shell: Rscript {0}
         run: |
           file_paths <- list.files("artifacts", include.dirs = FALSE, recursive = TRUE)
-          new_names <- paste0("binaries/", sub("/","-",file_paths))
+          new_names <- paste0("binaries/", sub("/","__",file_paths))

```

And likewise with L317?

```suggestion
          new_names <- file.path("binaries", sub("/", "__", file_paths))

```